src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -265,6 +272,16 @@ func (r *ConsoleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&appsv1.Deployment{}).
 		Owns(&corev1.Service{}).
 		Owns(&netv1.Ingress{}).
+		Watches(
+			&source.Kind{Type: &corev1.Pod{}},
+			handler.EnqueueRequestsFromMapFunc(r.reconcileConsoleForClusterPod),
+			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}),
+		).
```

Comment:
Just for my own understanding, and I'm not questioning your design decisions, I'm just trying to understand it better: what do we gain from watching the pods?

Reply:
Right, it should not be necessary. I'm double checking it.

The reason why I bound it to the pods is to trigger a reconciliation loop when a cluster starts forming a quorum, because the Console controller needs to do operations on the admin and Kafka APIs for creating a dedicated service account.

But there's a field in the Cluster CR that mirrors the ready state of the pods, which is `readyReplicas`, so it should be sufficient to watch the Cluster to trigger a reconcile loop at the right time.